### PR TITLE
Add mockups

### PR DESCRIPTION
**Pull request by decrn**
Proposal for #6.

### CODE REVIEW

File: docs/survey-builder.spec.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The final element of the stepper will be an "add category".
+The user selects the currently active category, which updates the area below to show the questions in this category.
+
+### Question builder
+
+The main builder area itself should present the user with a live preview of the question they're working on (taking up 70 percent of the screen),
+and the specifications pane to the right of it (taking up the remaining 30% of the screen) that contains all user editable properties of the active question:
+
+* The question type:
+    - range van 0 - X (rating)
```
Comment (kfrancois) [inline docs/survey-builder.spec.md]: A little 🇳🇱snuck in here

File: docs/survey-builder.spec.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Spec
```
Comment (kfrancois) [inline docs/survey-builder.spec.md]: What do you think about wiki-ing this stuff?

Comment (decrn) [reply docs/survey-builder.spec.md]: Sure. Whatever works. Checking it into version control just seemed more evident.

Comment (kfrancois) [reply docs/survey-builder.spec.md]: Wiki's are under version control, just a separate repo - check https://github.com/decrn/localsurvey.wiki.git

**Pull request merged**